### PR TITLE
Always bind invalidHandler, do so anonymously

### PR DESCRIPTION
**Pull request by YodaDaCoda**
This pr allows changing of invalidHandler after initialisation, and setting of invalidHandler if initialised without.

Fiddle:
https://jsfiddle.net/YodaDaCoda/fonmoyck/
Note that it's the initial invalidHandler that fires, not the replacement.

Fiddle with this patch:
https://jsfiddle.net/YodaDaCoda/Lugjrgge/
Note that the replacement invalidHandler fires.

### CODE REVIEW

File: src/additional/phoneAU.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Australian phone numbers have 10 digits (or 11 and start with +o31).
+ */
+$.validator.addMethod( "phoneNL", function( value, element ) {
```
Comment (staabm) [inline src/additional/phoneAU.js]: Doesnt belong into thos PR?

**Comment by staabm** [comment]
Whats your use case that requires this change?

File: src/additional/phoneAU.js (hunk 1 above)
Comment (YodaDaCoda) [inline src/additional/phoneAU.js]: Sorry, I'm pretty bad at git sometimes.

**Comment by YodaDaCoda** [comment]
I use the validator as part of a package which is deployed to clients with some default settings (the deployed code can't be changed, only added to). When clients want something that requires a modified invalidHandler, I encounter difficulty as I'm not able to change or remove the default action.

**Comment by staabm** [comment]
Sorry, but I dont think this change will progress the library into a good direction.

**Comment by staabm** [comment]
Will close for now. Feel free to ask further questions regarding this topic here. We can re-open if required.

**Pull request closed without merging**